src/test/java/emissary/test/core/junit5/ExtractionTest.java

```diff
@@ -171,8 +171,12 @@ protected void checkAnswers(Document answers, IBaseDataObject payload, List<IBas
     }
 
     protected void checkAnswers(Element el, IBaseDataObject payload, List<IBaseDataObject> attachments, String tname) throws DataConversionException {
-
         int numAtt = JDOMUtil.getChildIntValue(el, "numAttachments");
+        long numAttElements = el.getChildren().stream().filter(c -> c.getName().startsWith("att")).count();
+        if (Math.toIntExact(numAttElements) != numAtt && !(numAttElements <= 0 && numAtt == -1)) {
+            logger.warn("Expected numAttachments and actual <att#> count in {} not equal. Expected: {} Actual: {}", tname, Math.max(numAtt, 0),
```

Comment:
We have plenty of downstream answer files that only spot check among numerous attachments.  I should be worried if the number of `<attN>` elements exceeds the `<numAttachments>` value, but I'd hate to add hundreds of `<attN>` elements just to suppress this warning for a given test.  

Maybe we can add another element that allows "attN count validation" behavior to be overridden for the current answer file.  Something like `<attCountCheck>minimal</attCountCheck>` ?   (Naming things is hard.)

Reply:
I think that is something that could be added if that was the desire. I don't know the best way to go about it/if we'd want different logging levels if counts were excessively different, but the basic idea I agree could be helpful in those downstream environments.

Reply:
Another option (maybe better than the new answer file element) would be to add a private member to ExtractionTest that governs this behavior.  If desired, extending test classes could disable/alter the behavior by calling a setting that modifies the value of that private member -  e.g., `setAttachmentCountValidation(newValue);`

Reply:
That I think may end up being a cleaner/easier option in the long run. Would we only want to do this for attachments, or extracts as well?